src/components/molecules/MessageList.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="bg-gray-infolt p-6 space-y-6 overflow-auto"> 
+        <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+         <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+         <message-card 
+            timestamp="Weds, May 14th, 10:04 am" 
+            title="Your Employment Insurance Claim - You've been subscribed to Job Alerts" 
+            paragraphs="Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum ea eos ducimus nulla sit quae, commodi eaque ut eveniet laudantium amet maxime consectetur, facere repellendus ipsum tenetur numquam officia mollitia?"    
+         />
+        <message-card />
+        <message-card title="Empty box above" />
+    </div>
+<!--
+<div class="flex flex-col h-96 bg-gray-light p-6 space-y-6 overflow-auto"> 
+    <div class="bg-white p-6 space-y-3" v:for="msg of messages">
+        <span class="uppercase text-sm font-light">{{ msg.timestamp }}</span>
+        <h1 class="text-2xl font-semibold">{{ msg.title }}</h1>
+        <p class="font-normal text-lg">{{ msg.paragraphs }}</p>
+    </div>
```

Comment:
Clean up the comments if they are not needed :) Or leave a comment explaining what they are used for.